src/components/PlayerComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import PlayerLifeButton from './PlayerLifeButton';
+
+export class PlayerComponent extends Component {
+  state = {
+    currentLife: 40,
+    minLife: 0
+  };
+  addLife = () => {
+    this.setState((prevState, props) => ({
+      currentLife: prevState.currentLife + 1
+    }));
+  };
+
+  subtractLife = () => {
+    this.setState((prevState, props) => ({
+      currentLife: prevState.currentLife - 1
+    }));
+  };
+
+  render() {
+    const { store } = this.props;
+    return (
+      <View style={styles.playerContainer}>
+        <View style={styles.playerLifeContainer}>
+          <PlayerLifeButton
+            buttonText="+"
+            onPress={this.addLife}
+            currentLife={this.state.currentLife}
+          />
+          <Text style={styles.playerLifeText}>{this.state.currentLife}</Text>
+          <PlayerLifeButton
+            buttonText="-"
+            onPress={this.subtractLife}
+            currentLife={this.state.currentLife}
+          />
+        </View>
+        <TouchableOpacity
+          style={{
+            flex: 1,
+            flexDirection: 'row-reverse',
+            fontFamily: 'SourceCodePro-Bold'
+          }}
+        >
+          <View
+            style={{
+              alignContent: 'center',
+              justifyContent: 'center',
+              alignContent: 'center'
+            }}
+          >
```

Comment:
Pull these inline styles into the `styles` stylesheet?

Reply:
these will be changed in a future push